tests/foreman/api/test_virt_who_config.py

```diff
@@ -58,7 +62,10 @@ def test_positive_config_update(self):
         :id: 6b2cc2c3-959b-468b-9865-0f01decd2249
 
         :steps:
-            1. Update a config, verify it it changed and redploy it
+            1. Update a config,
```

Comment:
can you please add an example e.g. what config you would like to change along w/ file name ?